### DISCUSSION TITLE
Linking the graphics to templates/business-processes/Business Process Catalog

### DISCUSSION THREAD

**Discussion by SamkhawajaSKH** [microsoft/dynamics365patternspractices#118] [Ideas]
Hi,
This is a great start for establishing the guidelines. One suggestion is for the graphics, the process graphs/shapes, to indicate the business process catalog sequence with a hyperlink if possible, wherever applicable level 4 or level 3 i.e 10.20.010.100

![Screenshot 2023-11-30 111416](https://github.com/microsoft/dynamics365patternspractices/assets/152518824/30a18b1e-d393-4493-a150-51a1385255b4)
Thanks
Sam

**Comment by rachel-profitt** [comment]
Thanks for the suggestion. We will look to see if this is something others are wanting as well. As this may be difficult to maintain and update.
